### PR TITLE
{bio}[gompi/2021a] SRA-Toolkit v3.0.5, ncbi-vdb v3.0.5

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@SIB-software would it be possible to use the older version of `ncbi-vdb`? Or do you think it explicitly requires the newer one?

**Comment by smoretti** [comment]
I tried with the older version of `ncbi-vdb` but it failed (not the same lib names). Better if both ncbi-vdb and SRA-Toolkit are sync.
@boegel agreed I update the _test/easyconfigs/easyconfigs.py_ file in the PR to avoid the issue and "force" the sync.

**Comment by verdurin** [comment]
Fair enough - thanks for the update.

**Comment by smoretti** [comment]
In fact I am not allowed to update the `test/easyconfigs/easyconfigs.py` file:
`ERROR: Failed to process easyconfig ../../../../test/easyconfigs/easyconfigs.py: Failed to obtain class for ec.get('easyblock') easyblock (not available?): No module named 'ec'`

Could you do it @verdurin or @boegel ?
[easyconfigs.py.zip](https://github.com/easybuilders/easybuild-easyconfigs/files/11745631/easyconfigs.py.zip)

**Comment by smoretti** [comment]
I have created a distinct PR for the test file #18098

**Comment by boegel** [comment]
@smoretti Can you sync this with `develop` now #18098 is merged?

```
eb --sync-pr-with-devel 18098
```

File: easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-3.0.5-gompi-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+builddependencies = [
+    ('Perl',   '5.32.1', '-minimal'),
+    ('Python', '3.9.5', '-bare'),
+    ('CMake',  '3.20.1'),
+]
+
+dependencies = [
+    ('HDF5',    '1.10.7'),
+    ('libxml2', '2.9.10'),
+    ('bzip2',   '1.0.8'),
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-3.0.5-gompi-2021a.eb:26]:
```suggestion
    ('Perl', '5.32.1', '-minimal'),
    ('Python', '3.9.5', '-bare'),
    ('CMake', '3.20.1'),
]

dependencies = [
    ('HDF5', '1.10.7'),
    ('libxml2', '2.9.10'),
    ('bzip2', '1.0.8'),

File: easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-3.0.5-gompi-2021a.eb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+dependencies = [
+    ('Java',     '11',  '', SYSTEM),
+    ('OpenSSL',  '1.1', '', SYSTEM),
+    ('ncbi-vdb', version),
+    ('bzip2',    '1.0.8'),
+    ('file',     '5.40'),
+    ('HDF5',     '1.10.7'),
+    ('libxml2',  '2.9.10'),
+    ('zlib',     '1.2.11'),
```
Comment (boegel) [inline easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-3.0.5-gompi-2021a.eb:39]:
```suggestion
    ('CMake', '3.20.1'),
    ('flex', '2.6.4'),
    ('Bison', '3.7.6'),
    ('Perl', '5.32.1', '-minimal'),
    ('Python', '3.9.5',  '-bare'),
]

dependencies = [
    ('Java', '11',  '', SYSTEM),
    ('OpenSSL', '1.1', '', SYSTEM),
    ('ncbi-vdb', version),
    ('bzip2', '1.0.8'),
    ('file', '5.40'),
    ('HDF5', '1.10.7'),
    ('libxml2', '2.9.10'),
    ('zlib', '1.2.11'),
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-3.0.5-gompi-2021a.eb (hunk 2 above)
Comment (smoretti) [reply easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-3.0.5-gompi-2021a.eb]: done

File: easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-3.0.5-gompi-2021a.eb (hunk 1 above)
Comment (smoretti) [reply easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-3.0.5-gompi-2021a.eb]: done

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3143.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/0887c246e29d1299c2d33ac5509d6ded for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Going in, thanks @SIB-software!

**Pull request merged**